### PR TITLE
mimic: tests: make: *** [hello_world_cpp] Error 127 in rados

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/40320

### CODE REVIEW

**Comment by yuriw** [comment]
[wip-yuri6-testing-2019-07-23-2000-mimic](https://trello.com/c/PkgNbUWw/673-wip-yuri6-testing-2019-07-23-2000-mimic)

**Comment by yuriw** [comment]
@smithfarm see comments in https://trello.com/c/PkgNbUWw

File: qa/workunits/rados/test_envlibrados_for_rocksdb.sh [hunk 1]
```diff
@@ -40,15 +19,18 @@ CURRENT_PATH=`pwd`
...
+case $(distro_id) in
+	ubuntu|debian|devuan)
 		install g++ libsnappy-dev zlib1g-dev libbz2-dev librados-dev
 		;;
-	CentOS|Fedora|RedHatEnterpriseServer)
+	centos|fedora|rhel)
 		install gcc-c++.x86_64 snappy-devel zlib zlib-devel bzip2 bzip2-devel librados2-devel.x86_64
 		;;
+	opensuse*|suse|sles)
+		install gcc-c++ snappy-devel zlib-devel libbz2-devel libradospp-devel
```
Comment (tchaikov) [inline qa/workunits/rados/test_envlibrados_for_rocksdb.sh]: @smithfarm could you remove `libradospp-devel` from the list? as it does not exist in mimic.

File: qa/workunits/rados/test_librados_build.sh [hunk 2]
```diff
@@ -56,6 +58,14 @@ function run_binaries () {
 }
 
 pushd $DESTDIR
+case $(distro_id) in
+    centos|fedora|rhel|opensuse*|suse|sles)
+        install gcc-c++ make libradospp-devel librados-devel;;
```
Comment (tchaikov) [inline qa/workunits/rados/test_librados_build.sh]: ditto.

**Comment by yuriw** [comment]
[wip-yuri4-testing-2019-08-05-2052-mimic](https://trello.com/c/negGiBj1/685-wip-yuri4-testing-2019-08-05-2052-mimic)

**Comment by smithfarm** [comment]
Changelog:

* rebased
* dropped `libradospp-{dev,devel}` from the last two commits

@tchaikov Please take another look?

**Comment by yuriw** [comment]
[wip-yuri4-testing-2019-10-08-1529-mimic](https://trello.com/c/nS3vi0wl/712-wip-yuri4-testing-2019-10-08-1529-mimic)

**Pull request merged**